### PR TITLE
fix(license): update MIT license refs to BSD-3-Clause

### LINKED ISSUE #25: Looks like license is wrong
👋 As for the license file, and license field in cargo.toml, the project seems using MIT license, but the content in LICENSE-MIT.md is actually a BSD-3 body (also auto-detected by github as well). Might need a license change.

### PR DESCRIPTION
**Pull request by chenrui333**
fixes #25

### CODE REVIEW

**Comment by mrcnski** [comment]
Thanks! It looks like this was a pre-existing issue already present in `cargo-graph` when I had forked it.

According to the original license terms, the license text cannot be removed, so we should rather keep the license body and rename the file (and update Cargo.toml). Thanks for raising this!

**Comment by chenrui333** [comment]
cool, let me update to all refs to BSD-3-Clause then.

**Comment by chenrui333** [comment]
@mrcnski should be good now, thanks for the quick turnaround.

File: README.md [hunk 1]
```diff
@@ -102,11 +102,11 @@ It can be useful to exclude certain crates from the final graph. This can be ach
...
+For large dependency graphs, the `--no-transitive-deps` option can be used to filter out edges that are "covered" by a transitive dependency, which can make visual parsing a little easier by BSD-3-Clauseting some redundant edges. For example, if crate `A` depends directly on crate `B` and `C`, and crate `B` depends directly on crate `C`, this option would BSD-3-Clause the edge from `A` to `C`. To illustrate, compare the [default dependency graph](tokei-default.png) for Tokei, generated by `cargo deps`, to the [graph with transitive edges removed](tokei-transitive.png), generated by `cargo deps --no-transitive-deps`.
```
Comment (mrcnski) [inline README.md:109]: This is hilarious and I'm tempted to merge it in. 😄 But probably better to fix it. :)

**Comment by chenrui333** [comment]
you might need to do a new release as after this PR merge.

File: README.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ While `cargo-deps` should work for most use cases, it has an outdated architectu
 
 ## License
 
-`cargo-deps` is released under the terms of the MIT license. See the [LICENSE-MIT](./LICENSE-MIT) file for the details.
+`cargo-deps` is released under the terms of the BSD-3-Clause license. See the [LICENSE-BSD-3-Clause](./LICENSE-BSD-3-Clause) file for the details.
```
Comment (mrcnski) [inline README.md:146]: Maybe can just remove this section, seems redundant.

File: README.md (hunk 1 above)
Comment (chenrui333) [reply README.md:109]: oh sorry, I did the find-and-replace 😮‍💨

Comment (chenrui333) [reply README.md]: solved. :)

File: README.md (hunk 2 above)
Comment (chenrui333) [reply README.md:146]: you got it.

Comment (chenrui333) [reply README.md:146]: based on my experience, it is not that redundant though.

**Comment by chenrui333** [comment]
Let me know how it looks now.

File: README.md (hunk 2 above)
Comment (chenrui333) [reply README.md]: but indeed, less important than the GPL-3 license declarations.

**Review by mrcnski — APPROVED** [review]
Sweet, thanks!  I'll do a release right away.

**Comment by chenrui333** [comment]
that would be awesome!

**Comment by chenrui333** [comment]
so that I can refresh my homebrew pr 🎉

**Comment by mrcnski** [comment]
Done 🎉

**Pull request merged**